SourceryTests/Parsing/FileParser+SubscriptsSpec.swift

```diff
@@ -39,7 +39,7 @@ class FileParserSubscriptsSpec: QuickSpec {
                     expect(subscripts?[1]).to(equal(
                         Subscript(
                             parameters: [
-                                MethodParameter(argumentLabel: "b", name: "b", typeName: TypeName("Int"))
+                                MethodParameter(argumentLabel: "b", name: "b", typeName: TypeName("Int"), type: Type(name: "Int"))
```

Comment:
now when we have standard library types inferred does it still work properly when we have an extension of this type. It used to be the only case when we would set `type`, can you make sure we have a test for that and it still works?

Reply:
Sorry, I don't seem to fully understand what test do we need here. Can you expand on this a little bit?

Reply:
I probably meant something like what if we have an extension of Int in the scanned code? Will this type reference contain all symbols from this extension? Meaning do we set this type property to the fully resolved type?

Reply:
Got it, expanded unit test a little bit to have an extension on Int, and it's set for subscripts in more places then before(even though Int is reported as extension, because it's not seen anywhere else).